src/game.c

```diff
@@ -25,7 +25,7 @@ void player_name_inquiry(void);
 void start_new_game(void)
 {
     player_name_inquiry();
-
+    bool is_confirmed = false;
```

Comment:
Unused code, already deleted. Why add it back? Please update your branch with the latest code from development branch.

Reply:
Line 28 has been removed.